src/main/java/org/example/jvspringbootfirstbook/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package org.example.jvspringbootfirstbook.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotNull
         String title,
+        @NotNull
         String author,
+        @NotNull
         String isbn,
+        @NotNull
+        @Min(0)
         BigDecimal price,
+        @NotNull
         String description,
+        @NotNull
         String coverImage
```

Comment:
I think it would be better to use NotBlank annotation on all fields (except for price). Also there is a better annotation then `@Min(0)`.

```suggestion
        @NotBlank
        String title,
        @NotBlank
        String author,
        @NotBlank
        String isbn,
        @NotNull
        @PositiveOrZero
        BigDecimal price,
        @NotBlank
        String description,
        @NotBlank
        String coverImage
```